afkak/partitioner.py

```diff
@@ -1,53 +1,33 @@
 # -*- coding: utf-8 -*-
-# Copyright (C) 2015 Cyan, Inc.
-# Copyright 2017 Ciena Corporation
-
-import logging
+# Copyright 2015 Cyan, Inc.
+# Copyright 2017, 2018 Ciena Corporation
```

Comment:
Why?

Reply:
Oh, never mind. Brainfart. I didn't clue into the `import logging` being removed and thought you were just moving it below that line.